integration-cli/docker_cli_build_test.go

```diff
@@ -5753,15 +5751,20 @@ func (s *DockerTrustSuite) TestBuildContextDirIsSymlink(c *check.C) {
 
 // Issue #15634: COPY fails when path starts with "null"
 func (s *DockerSuite) TestBuildNullStringInAddCopyVolume(c *check.C) {
-	testRequires(c, DaemonIsLinux)
 	name := "testbuildnullstringinaddcopyvolume"
 
+	volName := "nullvolume"
+
+	if daemonPlatform == "windows" {
+		volName = `C:\\nullvolume`
+	}
+
 	ctx, err := fakeContext(`
 		FROM busybox
 
 		ADD null /
 		COPY nullfile /
-		VOLUME nullvolume
+		VOLUME `+volName+`
```

Comment:
I wonder if there's a really simple fix so that VOLUME commands don't have to be escaped with Windows filepaths. Not for this PR though.